src/components/Layout/Layout.js

```diff
@@ -46,6 +46,13 @@ const categoriesQuery = graphql`
         id
         name
         slug
+        relationships {
+          children {
```

Comment:
If no children categories are present, an error is thrown.
`/Users/smaharaj/git/gatsby-demo-store/src/components/Layout/Layout.js
  50:11  error  Cannot query field "children" on type "MoltinCategoryRelationships"  graphql/template-strings`

Anyway we can display subcategories only if they're present?

Reply:
@shaunmaharaj you're probably best extending the `MoltinCategory` GraphQL type to always have the `children` field, the best way to solve this.